opencog/nlp/relex2logic/rules/definite.scm

```diff
@@ -18,24 +19,23 @@
                 (VariableNode "$noun")
                 (VariableNode "$a-parse")
             )
-	    (InheritanceLink
-   		(VariableNode "$noun")
-   		(DefinedLinguisticConceptNode "definite")
-	    )
+            (InheritanceLink
+                (VariableNode "$noun")
+                (DefinedLinguisticConceptNode "definite")
+            )
         )
-       (ListLink
-        (ExecutionOutputLink
-       	   (GroundedSchemaNode "scm: pre-definite-rule")
-       	      (ListLink
-       	         (VariableNode "$noun")
+        (ListLink
+            (ExecutionOutputLink
+                (GroundedSchemaNode "scm: pre-definite-rule")
+                (ListLink (VariableNode "$noun"))
             )
         )
-      )
     )
 )
 
-; This is function is not needed. It is added so as not to break the existing
-; r2l pipeline.
+; This is function is not needed. It is added so as not to break the
+; existing r2l pipeline.  Huh ??? How can it not be needed?  It is
```

Comment:
OK, well, this seems to be done in multiple places in the code, and it does not seem to be obvious in any way.  The fix isn't clear to me.